src/tools/pcb_reader/main.cpp

```diff
@@ -521,7 +522,7 @@ int main( int argc, char* argv[] )
 
       printf( "Built export struct for %s in %lu seconds \n",
         zoneName.c_str(),
-        std::chrono::duration_cast< std::chrono::seconds >( std::chrono::high_resolution_clock::now() - entryStartTime ).count() );
+        static_cast<unsigned long>(std::chrono::duration_cast< std::chrono::seconds >( std::chrono::high_resolution_clock::now() - entryStartTime ).count() ));
```

Comment:
this static_cast is not ideal
prefer changing the format specifier %lu to match .count() type, for example %ld